src/gasx.f90

```diff
@@ -364,14 +364,14 @@ SUBROUTINE pistonvel(windspeed, Fice, N, kw660)
 
 ! INPUT variables
   !> wind speed at 10-m height
-  REAL(kind=r8), INTENT(out), DIMENSION(N) :: windspeed
+  REAL(kind=r8), INTENT(out) :: Fice
```

Comment:
Based on the comment below I figure this should be `Fice` and not `windspeed`. I also guess that `Fice` is not dimension of `N`.